### ISSUE TITLE
Exposure date

### ISSUE THREAD

**Issue by own2pwn** [CrunchyBagel/TracePrivately#45]
If we always initialize `ENTemporaryExposureKey` with the same `rollingStartNumber` of `0`,  do we get the actual timestamp of contact?

**Comment by HendX** [comment]
I don't know exactly how this works. The documentation states:

```
@property ENIntervalNumber rollingStartNumber;
This property contains the interval number when the key's TKRollingPeriod started.
```

But it doesn't describe what `ENIntervalNumber` nor what `TKRollingPeriod` is.

**Comment by dstotijn** [comment]
It's explained in the [crypto spec](https://covid19-static.cdn-apple.com/applications/covid19/current/static/contact-tracing/pdf/ExposureNotification-CryptographySpecificationv1.1.pdf) on page 6.

See https://github.com/dstotijn/exp-notif-crypto/blob/master/crypto.go#L64-L68 and https://github.com/dstotijn/exp-notif-crypto/blob/master/doc_test.go#L10-L20 for an example implementation in Go.

**Comment by HendX** [comment]
Thanks, this is all fixed in #49 and is now merged.
